app/types.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
```

Comment:
Is `types` descriptive enough for this (also remind me what these are, I think they're just meta data for Azure events, right?)